src/Checks/CacheHealthCheck.php

```diff
@@ -33,10 +33,11 @@ public function status(): Status
         foreach (config('healthcheck.cache.stores') as $store) {
             try {
                 $cache = Cache::store($store);
+                $key = 'laravel-health-check.' . random_bytes(32);
```

Comment:
This can generate non-ascii characters that may cause issues for caches, but thankfully we have `bin2hex()` to fix that

```suggestion
                $key = 'laravel-health-check.' . bin2hex(random_bytes(32));
```